wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -11,10 +11,10 @@ Members of the NAT are distinguished by their red username and user title. Like
 
 **All members of the Nomination Assessment Team are sworn to uphold [the Contributor Code of Conduct](/wiki/Contributor_Code_of_Conduct) in addition to the normal [Community Rules](/wiki/Rules).**
 
-If you encounter any member of the Nomination Assessment Team acting inappropriately, please contact [abuse@ppy.sh](mailto:abuse@ppy.sh) at once with as much information as possible. For more details, see [Reporting Abuse](/wiki/Reporting_Bad_Behaviour/Abuse).
-
 ## Responsibilities
 
+*Note: To report a member of NAT misbehaving or breaking the Contributor Code of Conduct, contact the [account support team](/wiki/People/The_Team/Account_support_team#support@ppy.sh).*\
```

Comment:
this one should be bolded&italic, or the same line in GMT article should be italic only